<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: red;}
QPushButton:Pressed {background-color: rgb(255, 100, 100);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(144, 238, 144)</string>
    </property>
    <property name="text">
     <string>Simon Says</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uniqueButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 213, 128)</string>
    </property>
    <property name="text">
     <string>Not Simon Says</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {background-color: blue;} 
QPushButton:Pressed {background-color: rgb(0, 191, 255);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="WinMessage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>-2</number>
    </property>
    <property name="text">
     <string>Good luck!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="InstructMessage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>-2</number>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
